src/data/faq.json

```diff
@@ -491,16 +491,30 @@
                         "active": true,
                         "textblock": [
-                            "If you see such a notice, your device is not able to send any random IDs via Bluetooth. This can be caused by a restriction due to your combination of hardware and software on your device. Because of it, the Corona-Warn-App cannot use Bluetooth Low Energy in its so-called “peripheral mode”.",
-                            "In case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. Your own risk assessment is <b>not affected</b>. All other features of the app are also not affected.",
-                            "Please ensure that you are using the latest version of the operating system available for your smartphone. If the problem persists, please contact your device manufacturer.",
-                            "If the card shows that your device is completely incompatible, the cause could be that your device does not contain a Bluetooth chip. Only in this case, you cannot exchange random IDs with others and risk assessment can only base its calculation on locations and events you checked in to.",
-                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction in any case</b> and only proximity-based ID recording is affected.",
-                            "Bluetooth compatibility checking and warnings were introduced with Android Corona-Warn-App version 2.2."
+                            "Since version 2.2, the Corona-Warn-App for Android contains a check which determines whether your device is fully capable of all Bluetooth-related features which the app needs to operate normally. If you are seeing this message, the operating system of your device is lacking a functionality that Corona-Warn-App normally uses.",
+                            "Due to this, your device is not able to <i>send</i> any random IDs via Bluetooth. However, you are still able to <i>receive</i> random IDs. Therefore, <b>your own risk assesment is not affected</b>.",
+                            "The only limitation which this incompatibility causes is that, in case you are infected and you decide to warn others, only users who were checked in to the same event or the same location, at the same time as you were, can be warned. All other features of the app are not affected.",
+                            "To resolve the problem, please ensure that you are using the latest version of the operating system available for your smartphone. If this doesn't help, please contact your device manufacturer.",
+                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction</b> and only proximity-based ID recording is affected.",

```

Comment:
I'd say "are fully functional", since we're talking about more than one feature, or?

Reply:
From my feeling, since I'm separating the first batch and "retrieving […] test results" through an "as well as", the verb would grammatically refer only to the test result feature, but honestly I have no idea how to find out which version is correct :sweat_smile:

Reply:
Maybe @MikeMcC399 would like to help out here? 🙂

Reply:
This sentence seems redundant, because the message has already been given in L500. 

Grammatically it is a little awkward because the main sentence says:
"Both Check-in and Contact Journal functionality is fully functional without restriction and only proximity-based ID recording is affected."
functionality / functional appears twice and "fully" and "without restriction" is a repeat.

I would tend to just drop this sentence.

Reply:
@heinezen suggested to add this information twice in https://github.com/corona-warn-app/cwa-website/pull/1211#discussion_r627500614.

@MikeMcC399, I think you are right with considering the sentence without the additional sentence part starting with "as well as", and then it needs to be "Both […] are".